CONTRIBUTING.md

```diff
@@ -2,38 +2,39 @@
 
 ## This project was made for first-time contributors and open source beginners
 
-This project follows the [Poppins manifesto guidelines](github.com/bancodobrasil/poppins/) as part of it's community principles and policies, focusing all the decisions and interactions on providing open source beginners mentorship with real and relevant experiences, respecting each learning pace, background experience, academic formation, questions, suggestions, doubts and opinion.
+This project follows the [Poppins manifesto guidelines](github.com/bancodobrasil/poppins/) as part of it's community principles and policies, focusing all the decisions and interactions on providing open source beginners mentorship with real and relevant experiences, respecting each learning pace, background experience, academic formation, questions, suggestions, doubts and opinions.
 Contribute now!
 
-So, **let's start contributing! Open an issue asking for a task to be done by you**. A mentor/maintainer will come and provide a technical overview of the project and what are the possibles ways of contributing to the project. You will discuss the options and a suitable issue will be assigned or created to you.
+So, let's start contributing! **First, go to the 'Issues' tab and open a new issue, titled something like: "I want to contribute." Then, in the first comment, introduce yourself and ask for a task to be assigned to you**. If you already have an issue in mind, feel free to describe it in that same comment. A mentor/maintainer will respond with a technical overview of the project and some possible ways of contributing. You will discuss the options, and when a suitable issue is agreed upon, the mentor will create a new issue and assign it to you. 
```

Comment:
Nice, @TCashion !
What do you think about making a PR to https://github.com/bancodobrasil/poppins also with this and other suggestions?